hedera-node/src/main/java/com/hedera/services/grpc/marshalling/AdjustmentUtils.java

```diff
@@ -55,14 +57,14 @@ public static BalanceChange adjustedChange(
 		final var extantChange = manager.changeFor(account, denom);
 		if (extantChange == null) {
 			if (denom == MISSING_ID) {
-				final var newHbarChange = BalanceChange.hbarAdjust(account, amount);
+				final var newHbarChange = BalanceChange.hbarAdjust(account, amount, DEFAULT_PAYER, DEFAULT_ALLOWANCE_APPROVAL);
```

Comment:
I might be nice to have an overloaded version of the `BalanceChange.hbarAdjust(account, amount)` factory that fills in these defaults when the caller doesn't supply them. (Same for the `tokenAdjust()` factory.)